tools/precommit.py

```diff
@@ -206,6 +206,7 @@ def get_os_dependency_exclude_module(exclude_module):
             build(buildtype, ['--target-arch=arm',
                               '--target-os=tizen',
                               '--target-board=artik10',
+                              '--compile-flag=-D__LINUX__',
```

Comment:
See change in iotjs.spec. It introduces __TIZEN__ flag there, which enables usage of Peripheral Tizen API.